File: nunit.templates/ItemTemplates/Test/NUnit Test Fixture/TestClass.cs
// NUnit3 Tests,  see documentation : https://github.com/nunit/docs/wiki/NUnit-Documentation 
using NUnit.Framework;

namespace $rootnamespace$
{
    [TestFixture]
    public class $safeitemname$
    {
        [Test]
        public void TestMethod()
        {
            // TODO: Add your test code here
            var answer = 42;
            Assert.That(answer,Is.EqualTo(42),"Some useful error message");
        }
    }
}

File: nunit.templates/ItemTemplates/Test/NUnit Test Fixture/TestClass.vb
Option Explicit On

Imports System
Imports NUnit.Framework

Namespace $rootnamespace$

    <TestFixture>
    Public Class $safeitemname$

        <Test>
        Public Sub TestMethod()
            ' TODO Add your test code here
            Dim answer = 42
            Assert.That(answer, [Is].EqualTo(42), "Some useful error message")
        End Sub

    End Class

End Namespace